build.sbt

```diff
@@ -226,15 +226,18 @@ lazy val `integration-test-local` = project
 
 lazy val docs = project
   .in(file("docs"))
-  .enablePlugins(ParadoxPlugin, NoPublish)
+  .enablePlugins(AkkaParadoxPlugin, NoPublish)
   .disablePlugins(BintrayPlugin)
   .settings(
     name := "Akka Management",
-    paradoxGroups := Map("Language" -> Seq("Scala", "Java")),
-    paradoxTheme := Some(builtinParadoxTheme("generic")),
-    paradox in Compile := (paradox in Compile).dependsOn(LocalRootProject / ScalaUnidoc / doc).value,
-    paradoxProperties ++= Map(
+    publish / skip := true,
+    whitesourceIgnore := true,
+    paradoxGroups := Map("Language" -> Seq("Java", "Scala")),
```

Comment:
Do we really want Java first? OK by me, but it's not what we do for most other projects AFAIK

Reply:
This is part of the roll-out :-)